codecov.yml

```diff
@@ -1,20 +1,32 @@
-comment: true
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: default
+  require_changes: true    # only post the comment if coverage changes
 ignore:
   - "l2geth"
   - "**/*.t.sol"
   - "op-bindings/bindings/*.go"
-  - "packages/contracts-bedrock/contracts/(vendor|libraries)/(trie|rlp)"
+  - "packages/contracts-bedrock/contracts/vendor"
+  - "packages/contracts-bedrock/contracts/libraries/trie"
+  - "packages/contracts-bedrock/contracts/libraries/rlp"
```

Comment:
AFAICT the LHS is correct, but it seems to have been causing everything in `packages/contracts-bedrock/contracts/` to be ignored.